flamedisx/er_nr_base.py

```diff
@@ -305,7 +318,7 @@ def _differential_rate(self, data_tensor, ptensor):
 
         p_ph = self.detection_p('photon', data_tensor, ptensor)
         p_el = self.detection_p('electron', data_tensor, ptensor)
-        d_ph = self.detector_response('photon', data_tensor, ptensor)
+        d_ph = self.detector_response('photon', data_tensor, ptensor) # p(s1|ph_detected)
```

Comment:
Thanks for adding the comment. You could consider putting it on a separate line (so the line itself stays within 80char), and /or adding similar comments for the other three similar lines?